Tasks/ManualValidationV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -7,6 +7,8 @@
   "loc.input.help.notifyUsers": "Send a manual validation pending email to specific users (or groups). Only users specified in approvers can act on a manual validation.",
   "loc.input.label.approvers": "Approvers",
   "loc.input.help.approvers": "Specify users/groups/project teams separated by commas to act on a manual validation. In absence of input, users with queue build permission will be able to take action.",
+  "loc.input.label.allowApproversToApproveTheirOwnRuns": "Allow approvers to approve their own run",
+  "loc.input.help.allowApproversToApproveTheirOwnRuns": "If this is true, approver will be able to approve their own run",
```

Comment:
> Documentation changes required: (Y/N) Y

How this will be covered? Do we have WI for that?

Reply:
For updating the documentation on new changes, [this repo](https://github.com/MicrosoftDocs/azure-devops-yaml-schema/blob/main/task-reference/manual-validation-v1.md) will be updated.

Reply:
> [this repo](https://github.com/MicrosoftDocs/azure-devops-yaml-schema/blob/main/task-reference/manual-validation-v1.md) will be updated.

(process-wise) how? by whom?

Reply:
doc changes will updated by Steve Danielson. WI to track this linked in ADO PR.